src/components/PatchesPage/patchCard/DropdownMenu.tsx

```diff
@@ -17,14 +17,14 @@ interface Props {
   childPatches: Partial<Patch>[];
   hasVersion: boolean;
 }
-export const DropdownMenu: React.FC<Props> = ({
```

Comment:
Just flagging that we probably want to go with `React.VFC` here

Reply:
Thats covered in my other pr. :) Its not yet merged.

Reply:
True, just didn't know if retyping to `JSX.Element` was appropriate here!

Reply:
Oh good catch. I'll make sure its updated when i merge in main in the other pr.